dash/test/dart/DARTMemAllocTest.cc

```diff
@@ -109,7 +109,7 @@ TEST_F(DARTMemAllocTest, LocalAlloc)
 
 TEST_F(DARTMemAllocTest, SegmentReuseTest)
 {
-  const size_t block_size = 10;
+  const size_t block_size = 10*dash::size();
```

Comment:
In this test the memory is allocated through `dart_team_memalloc_aligned` so it's not necessary to scale the size with the number of units.

Reply:
Fixed it.